flybywiresim/aircraft
# feat(pfd): add true bearing flag & refactor init

## Summary of Changes


Adds on the A380X PFD the TRUE heading label when TRUE heading mode is active.

Thanks @tracernz for the assistance and refactoring the initialization. 
In another PR, the same change to the A32NX should be done.  

## References





To verify the TRUE is the same as the other aircrafts (specifically the A320, as this is using A320 code)
A380 ref: https://youtu.be/XQuXdOlZbNE?si=OFthTootiKYr_Mhg&t=19
A350 ref: https://youtu.be/qmX0_-9Ggmw?si=LhRs4sILSh_zK_nr&t=543

Thanks @tracernz! 

## Additional context



Discord username (if different from GitHub): alepouna

## Testing instructions


A32NX: 
- Check for any issues in the PFD
- Verify TRUE heading labels show up (Set the `A32NX_PUSH_TRUE_REF` LVAR via Behaviors to 1) 

A38X: 

- Set the `A32NX_PUSH_TRUE_REF` LVAR via Behaviors to 1
- Observe PFD shows TRUE box
- Check for any issues in the PFD 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
